RealmSwift-swift2.2/Tests/RealmTests.swift

```diff
@@ -255,6 +255,29 @@ class RealmTests: TestCase {
         }
         XCTAssertEqual(try! Realm().objects(SwiftStringObject.self).count, 0)
     }
+    
+    func testThrowsWrite() {
+        enum TestError: ErrorType { case Exception }
```

Comment:
I threw `Realm.Error.Fail` in the test, but I'm not too happy with that as the test could erroneously succeed. Any suggestions @austinzheng, @tgoyne?